app/views/CartPage/index.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useCallback, useState } from 'react';
+import { Button, Container, NumberInput, TextOutput } from '@the-deep/deep-ui';
+import {
+    gql,
+    useMutation,
+    useQuery,
+} from '@apollo/client';
+import { AiTwotoneDelete } from 'react-icons/ai';
+import { FaHeart } from 'react-icons/fa';
+
+import styles from './styles.css';
+import { CartListQuery, CartListQueryVariables, DeleteCartItemsMutation, DeleteCartItemsMutationVariables } from '#generated/types';
+
+const CART_LIST = gql`
+    query CartList ($email: ID!, $page: Int!, $pageSize: Int!) {
+        cartItems (createdBy: $email, page: $page, pageSize: $pageSize){
+            results {
+                id
+                totalPrice
+                book {
+                id
+                title
+                image {
+                    url
+                }
+                authors {
+                    id
+                    name
+                }
+                price
+                }
+                quantity
+            }
+            grandTotalPrice
+            page
+            pageSize
+            totalCount
+        }
+    }
+`;
+
+const DELETE_CART_ITEMS = gql`
+mutation DeleteCartItems ($id: ID!) {
+    deleteCartItem(id: $id) {
+      errors
+      ok
+    }
+  }
+`;
+
+interface Author {
+    id: number;
+    name: string;
+}
+
+interface Image {
+    url: string;
+}
+
+interface Book {
+    id: number;
+    price: number;
+    authors: Author[];
+    quantity: number;
+    image: Image;
+    title: string;
+    cartId: string;
+    removeBookFromCartList: (id: string) => void;
+}
+
+function CartContent(props: Book) {
+    const { id, price, authors, image, title, quantity, removeBookFromCartList, cartId } = props;
+    const authorsDisplay = React.useMemo(() => (
+        authors?.map((d) => d.name).join(', ')
+    ), [authors]);
+
+    const deleteCartItem = useCallback(() => {
+        removeBookFromCartList(cartId);
+    }, [cartId]);
```

Comment:
set name of button to cartId and directly set removeBookFromCartList in onClick.